app/controllers/test/components/mentoring/queue_controller.rb

```diff
@@ -41,12 +43,13 @@ def solutions
         have_mentored_previously: true,
         status: "First timer",
         updated_at: 1.week.ago.iso8601,
-        url: "https://exercism.io/solutions/3"
+        url: "https://exercism.io/solutions/3",
+        tooltip_url: Exercism::Routes.student_test_components_tooltips_tooltip_path(3)
       }
     ]
 
     page = params.fetch(:page, 1).to_i
-    per = params.fetch(:per, 1).to_i
+    per = params.fetch(:per, 2).to_i
```

Comment:
This makes testing the relative positioning of the tooltips easier. I'll revert later.